README.md

```diff
@@ -105,15 +105,31 @@ always in sync, and that one or the other may be slightly more up-to or out-of d
 
 ### `review.tsv`
 Columns:
-- `classCode`: integer: ID of review case class
-- `classShortName`: string (camelCase): describing the review case class
+- `classCode`: integer
+- `classLabel`: string
 - `value`: any: Some form of data to review
 - `comment`: string (optional)
 
-#### 1. `causalD2gButMarkedDigenic`
-This review case involves what would be otherwise considered a valid disease-gene relationship, but for the fact that 
-it quite unusually includes 'digenic' in the label, even though it only had 1 association. OMIM doesn't have a 
+#### 1. D2G Disease-defining but marked digenic
+This review case involves what would be otherwise considered a valid disease-gene (D2G) relationship, but for the fact 
+that it quite unusually includes 'digenic' in the label, even though it only had 1 association. OMIM doesn't have a 
 guaranatee on the data quality of its disease-gene associations marked 'digenic', so for any of these entries, it could 
 be the case that either (a) it is not 'digenic'; OMIM should remove that from the label, and Mondo can make an explicit 
 exception to add the relationship, or could otherwise wait until OMIM fixes the issue and it will automatically be 
 added, or (b) it is in fact 'digenic', and OMIM should add the missing 2nd gene association.
+
+#### 2. D2G: Disease-defining; self-referential
+The unique characteristics of cases of this class are as follows: 
+- Each case has 2 rows in `morbidmap.txt` and are related.
+- Row 1: One row is a typical, valid, disease-defining entry. For the given phenotype MIM in that row, there are no 
```

Comment:
There is already the link to the [google sheet](https://docs.google.com/spreadsheets/d/1hKSp2dyKye6y_20NK2HwLsaKNzWfGCMJMP52lKrkHtU/edit?gid=1826025296#gid=1826025296). I think that's probably enough.

However, the formatting of the spreadsheet is not 100% the same as `morbidmap.txt`, and it can be hard to interrogate that file sometimes.

So I added this table as an example:
|Phenotype|Gene/Locus And Other Related Symbols|MIM Number|Cyto Location|
|-|-|-|-|
|Small cell cancer of the lung, somatic, 182280 (3)|RB1|614041|13q14.2|
|Small-cell cancer of lung (2)|SCLC1|182280|3p23-p21|

I think this will be sufficient to show how / where the Phenotype appears.

Reply:
Looks great! Thanks!